package.json

```diff
@@ -22,6 +22,7 @@
     "@types/webdriverio": "^4.7.0",
     "chai": "^4.1.0",
     "chai-string": "^1.4.0",
+    "colors": "^1.4.0",
```

Comment:
What is the motivation behind including `colors` in the pull-request?
Is it solely used as a visual aid during debugging?

Please register a `devDependency` CQ for the following if it is not already covered:
https://github.com/eclipse-theia/theia/wiki/Registering-CQs#case-new-or-updated-testbuild-dependency-eg-devdependencies

Reply:
It is used for debugging, it really helps making sense of the different streams produced during the test execution. Will check it out following your link.